src/test/java/com/google/sps/SyncServletTest.java

```diff
@@ -0,0 +1,105 @@
+        Room testRoom = new Room(new LinkedList<Member>());
+        testRoom.addVideo(Video.createVideo("www.RossJohnson.com"));
+        String expected = "{\"currentState\":\"UNSTARTED\",\"currentVideoTimestamp\":0,\"id\":\"www.RossJohnson.com\"}";
+        
+        String actual = SyncServlet.doGetHelper(testRoom);
+        
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testVideoUpdateTimeStamp() {
+        Video.VideoState newState = Video.VideoState.PLAYING;
+        Room room = new Room(new LinkedList());
+        room.addVideo(Video.createVideo("This is a url"));
+        long expected = testTimestamp;
+        
+        room.updateCurrentVideoState(newState, testTimestamp);
+        long actual = room.getVideos().peek().getCurrentTimeStamp();
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testVideoUpdateState() {
+        Video.VideoState newState = Video.VideoState.PLAYING;
+        Room room = new Room(new LinkedList());
+        room.addVideo(Video.createVideo("This is a url"));
+        Video.VideoState expected = newState;
+
+        room.updateCurrentVideoState(newState, testTimestamp);
+        Video.VideoState actual = room.getVideos().peek().getCurrentState();
+        
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testVideoSyncWithVideoEnded() {
+        HttpServletRequest req = Mockito.mock(HttpServletRequest.class);
+        HttpServletResponse res = Mockito.mock(HttpServletResponse.class);
+        //Room that is being passed through the helper function
+        Room room = new Room(new LinkedList<Member>());
+        room.addVideo(Video.createVideo("www.Ross.com"));
+        room.addVideo(Video.createVideo("www.Johnson.com"));
+        String expected = "{\"currentState\":\"UNSTARTED\",\"currentVideoTimestamp\":0,\"id\":\"www.Johnson.com\"}";
+        
+        SyncServlet.updateRoomVideos(room, Video.VideoState.ENDED, 0);
+        String actual = SyncServlet.roomToVideoJson(room);
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testVideoSyncWithVideoOngoing() {
+        HttpServletRequest req = Mockito.mock(HttpServletRequest.class);
+        HttpServletResponse res = Mockito.mock(HttpServletResponse.class);
+        //Room that is being passed through the helper function
+        Room room = new Room(new LinkedList<Member>());
+        room.addVideo(Video.createVideo("www.Ross.com"));
+        room.addVideo(Video.createVideo("www.Johnson.com"));
+        //Room to have the expected properties when we test
+        String expected = "{\"currentState\":\"PAUSED\",\"currentVideoTimestamp\":1234,\"id\":\"www.Ross.com\"}";
+        
+        SyncServlet.updateRoomVideos(room, Video.VideoState.PAUSED, 1234);
+        String actual = SyncServlet.roomToVideoJson(room);
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testVideoQueueEmpty(){
```

Comment:
testDoGetHelper_videoQueueEmpty. Rename the test method to be as testFunctionName_testCase pattern.

Update other functions' name too please.